packages/kmod-6.1-nvidia/nvidia-sysusers.conf

```
@@ -0,0 +1 @@
+u nvidia - "nvidia-persistenced user"
```

nit: since we might use it for something else because the username isn't `persistenced` specific:
```suggestion
u nvidia - "nvidia user"
```